python/src/aiconfig/editor/server/server.py

```diff
@@ -109,27 +110,88 @@ def _get_server_state(app: Flask) -> ServerState:
 
 @app.route("/")
 def home():
-    ss = _get_server_state(app)
-    ss.count += 1
-    LOGGER.info("Count: %s", ss.count)
     return app.send_static_file("index.html")
 
 
-@app.route("/test")
-def test():
-    return {"key": 6}
+@app.route("/api/load", methods=["POST"])
+def load():
+    ss = _get_server_state(app)
+    request_json = request.get_json()
+    path = request_json["path"]
+    LOGGER.info(f"Loading AIConfig from {path}")
+    if not path:
+        return {"message": "No path provided"}, 400
+    elif not os.path.isfile(path):
+        return {"message": f"File does not exist: {path}"}, 400
+    else:
+        try:
+            ss.aiconfig_runtime = AIConfigRuntime.load(path)  # type: ignore
+            return {"message": "Done"}, 200
+        except Exception as e:
+            return {"message": f"<p>Failed to load AIConfig from {path}: {e}"}, 400
+
+
+@app.route("/api/save", methods=["POST"])
+def save():
+    ss = _get_server_state(app)
+    request_json = request.get_json()
+    path = request_json["path"]
```

Comment:
missing `include_outputs`. I think it would be much better to stick close to the properties/data types expected by AIConfig API. Makes it easy to use this server